docs/details/advisors/checks/Develop Security Checks

```diff
@@ -0,0 +1,260 @@
+The Security Threat Tool in PMM offers a set of checks that can detect common security threats, performance degradation, data loss and data corruption. 
+
+As a developer, you can create custom checks to cover additional use cases that are relevant to your specific database infrastructure. 
+
+
+## Check components
+A check is a combination of:
+
+- SQL query or MongoDB query document for extracting data from the database;
+- Python script for converting extracted data into check results. This is actually a [starlark-go](https://github.com/google/starlark-go) script – [an extension](https://github.com/google/starlark-go/blob/master/doc/spec.md#dialect-differences) of the [original Starlark language](https://docs.bazel.build/versions/main/skylark/language.html) that adds more imperative features from the Python. Script's execution environment is sandboxed; no I/O can be done from it.
```

Comment:
Probably better to write `Starlart script (Python dialect)...`

Reply:
I also would like to introduce the concept of Advisors here. So our scope is (when features will be in) to develop few Advisors that use N checks to review a node/platform and provide contextualized suggestions. 
Another thing is that we are talking here (filename) of Security check (security TT) but in reality we moved out from that, and we are now including STT as one the security Advisor (which has multiple security checks)

Reply:
actually we should have more given our discussions:
```
    #family: mongodb
    #advisor_type: configuration
    #advisor_group: database config advisor
```
for instance, this because we are moving from single check to advisor concept.